### DISCUSSION TITLE
The requested URL returned error: 403 Forbidde

### DISCUSSION THREAD

**Discussion by Srivally27** [community/community#73206] [New to GitHub]
RPC failed; HTTP 403 curl 22 The requested URL retured error: 403
fatal: expected 'acknowlegments'

**Comment by queenofcorgis** [comment]
Hi @Srivally27 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.
